src/components/chart/ha-chart-base.ts

```diff
@@ -450,7 +450,7 @@ export class HaChartBase extends LitElement {
         border-radius: 4px;
         pointer-events: none;
         z-index: 1000;
-        width: 200px;
+        width: 230px;
```

Comment:
Why was the width changed? Now the positioning logic will not be correct anymore.

Reply:
I see, you mean "_handleTooltip()" probably. My fault. Thought about increasing the width a little.
Trying to solve it here:
```
        clamp(
          context.tooltip.caretX,
          100,
          this.clientWidth - 100 - this._paddingYAxisInternal
        ) -
        100 +
        "px",
```
with no success so far...

Reply:
Restored width to 200px. Will try to sort it out later then(((. Sorry, still doing 1st steps.